src/python/library/tritonclient/http/__init__.py

```diff
@@ -632,7 +633,7 @@ def load_model(self,
                    headers=None,
                    query_params=None,
                    config=None,
-                   encoded_files={}):
+                   files={}):
```

Comment:
Empty dictionaries in the default argument can behave strangely. It is not an issue here since we are not modifying the dictionary but just something to be aware of:

https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python